var/spack/repos/builtin/packages/xios/package.py

```diff
@@ -42,6 +42,9 @@ class Xios(Package):
     patch("llvm_bug_17782.patch", when="@1.1: %apple-clang")
     patch("llvm_bug_17782.patch", when="@1.1: %clang")
 
+    # This patch fixes a compilation error where 'FLT_MAX' and 'FLT_MIN' was not declared in scope.
+    patch("fix_float_error.patch", when="@develop")
```

Comment:
`@develop` is meant to be a moving target, there's really little point having a patch for it - the patch will hopefully be outdated soon if the `develop` branch doesn't compile.

Reply:
@alalazo 
I have created a version "develop-2612" with a fixed revision of develop and patched to it. I need the develop branch and this patch, not version 2.5, to build the [nemo package](https://github.com/spack/spack/pull/42306).